### PR TITLE
fix bug in #695

### PR DESCRIPTION
**Pull request by realwanpengli**

### CODE REVIEW

**Comment by BrennanConroy** [comment]
What bug? Where's the test(s)?

**Comment by realwanpengli** [comment]
Tests are in `ServiceHubContextE2EFacts`. the properties in `HubOptions` are null for `ServiceManager`.

File: src/Microsoft.Azure.SignalR.Management/ManagementHubOptionsSetup.cs [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+        internal const int DefaultStreamBufferCapacity = 10;
+
+        private readonly List<string> _defaultProtocols = new List<string>();
+
+        public ManagementHubOptionsSetup(IEnumerable<IHubProtocol> protocols)
+        {
+            foreach (var hubProtocol in protocols)
+            {
+                if (hubProtocol.GetType().CustomAttributes.Where(a => a.AttributeType.FullName == "Microsoft.AspNetCore.SignalR.Internal.NonDefaultHubProtocolAttribute").Any())
```
Comment (vicancy) [inline src/Microsoft.Azure.SignalR.Management/ManagementHubOptionsSetup.cs]: you don't expose DI, so you don't need this?

Comment (vicancy) [reply src/Microsoft.Azure.SignalR.Management/ManagementHubOptionsSetup.cs]: How is this resolved?

Comment (realwanpengli) [reply src/Microsoft.Azure.SignalR.Management/ManagementHubOptionsSetup.cs]: failed to git push, updated

File: src/Microsoft.Azure.SignalR.Management/ManagementHubOptionsSetup.cs [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+
+        internal const int DefaultMaximumMessageSize = 32 * 1024;
+
+        internal const int DefaultStreamBufferCapacity = 10;
+
+        private readonly List<string> _defaultProtocols = new List<string>();
+
+        public ManagementHubOptionsSetup(IEnumerable<IHubProtocol> protocols)
+        {
+            foreach (var hubProtocol in protocols)
```
Comment (vicancy) [inline src/Microsoft.Azure.SignalR.Management/ManagementHubOptionsSetup.cs]: Who is adding the `IHubProtocol`s in Management API, and what is the current value?

**Pull request merged**